README.md

```diff
@@ -64,50 +64,46 @@ run()
 
 See the [toolkit documentation](https://github.com/actions/toolkit/blob/master/README.md#packages) for the various packages.
 
-## Publish to a distribution branch
+## Package for distribution
 
-Actions are run from GitHub repos.  We will create a releases branch and only checkin production modules (core in this case). 
+Actions are run from GitHub repos.  Packaging the action will create a self encapsulated distribution runnable from the GitHub graph.
 
 Comment out node_modules in .gitignore and create a releases/v1 branch
 ```bash
-# comment this out distribution branches
-# node_modules/
+npm run package
 ```
 
+Since the packaged index.js is run from the dist folder, check it in.
+
 ```bash
-$ git checkout -b releases/v1
-$ git commit -a -m "prod dependencies"
+git add dist
+```
+
+## Create a release branch
+
+Users shouldn't consume the action from master since that would be latest code and actions can break compatibility between major versions.
+
+```bash
+$ git checkout -b v1
+$ git commit -a -m "v1 release"
 ```
 
 ```bash
-$ npm prune --production
-$ git add node_modules
-$ git commit -a -m "prod dependencies"
 $ git push origin releases/v1
 ```
 
 Your action is now published! :rocket: 
 
 See the [versioning documentation](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md)
```

Comment:
I realized this versioning doc needs to get rewritten in light of this.  Will do that next and update this link.